packages/expo-notifications/plugin/src/__tests__/withNotificationsAndroid-test.ts

```diff
@@ -86,14 +84,7 @@ describe('Android notifications configuration', () => {
       '#123456'
     );
   });
-  it('writes to colors.xml correctly', async () => {
```

Comment:
would be nice to still test this behavior, even though now we're using `withAndroidColors`

Reply:
Don't really need to test this behavior as it's now essentially assigning a value to an object using a tested mod and a tested method.